src/main/scala/uk/gov/hmrc/play/http/HeaderCarrier.scala

```diff
@@ -114,10 +120,25 @@ object HeaderCarrier {
       headers.get(HeaderNames.trueClientPort),
       headers.get(HeaderNames.googleAnalyticTokenId),
       headers.get(HeaderNames.googleAnalyticUserId),
-      getDeviceId(cookies, headers)
+      getDeviceId(cookies, headers),
+      headers.get(HeaderNames.akamaiReputation).map(AkamaiReputation),
+      remainingHeaders(headers)
     )
   }
 
+  val blacklistedHeaders: Seq[String] = {
+    // TODO sensible config key name and default values
```

Comment:
Any suggestions for sensible config key names gratefully received! Since this config is currently hypothetical and would be a platops concern, I was a little coy about naming it ...